src/components/LicenseEntity/LicenseMatchesTable.tsx

```diff
@@ -53,6 +55,18 @@ const LicenseMatchesTable = (props: LicenseMatchProps) => {
                     </td>
                   </tr>
                 )}
+                {match.from_file && (
+                  <tr>
+                    <td colSpan={2}>Matched file</td>
```

Comment:
```suggestion
                    <td colSpan={2}>From file</td>
```
Let's keep this same as the attribute to avoid confusion.

Reply:
Updated :)